CODE-GUIDELINES.md

```diff
@@ -37,3 +37,59 @@ test('...', () => {
   vi.mocked(window.windowMethod).mockResolvedValue('a string');
 });
 ```
+
+### Mock before tests, not during tests
+
+When you mock functions for some tests, it is recommended to define the mocks from the `beforeEach` callback inside a `describe`,
```

Comment:
It really depends on the test, if all the tests share the same mocked value, yes, if not, it is easier for a reader to see the specific mocked value inside the test